packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFailedPaymentsCancellation/en.ftl

```diff
@@ -0,0 +1,7 @@
+#  Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionFailedPaymentsCancellation-title = Your subscription has been cancelled
+subscriptionFailedPaymentsCancellation-subject = Your <%- productName %> subscription has been cancelled
+#  Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionFailedPaymentsCancellation-content = We've cancelled your { $productName } subscription because multiple payment attempts failed. To get access again, start a new subscription with an updated payment method.
```

Comment:
Sorry, I forgot to mention that the email templates use [curly quotes] instead of straight quotes.(https://typographyforlawyers.com/straight-and-curly-quotes.html) Please update the apostrophe in `We've` to use a curly closing single quote.